docs/core/entity/device-tracker.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Device Tracker Entity
+sidebar_label: Device Tracker
+---
+
+A device tracker is a read-only entity that provides presence information. There are two types of device tracker entities, a ScannerEntity and a TrackerEntity.
+
+## ScannerEntity
+
+A ScannerEntity reports the connected state of a device on the local network. If the device is connected the ScannerEntity will have state `home` and if the device is not connected the state will be `not_home`.
+
+### Properties
+
+:::tip
+Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+:::
+
+| Name          | Type    | Default      | Description                                       |
+| ------------- | ------- | ------------ | ------------------------------------------------- |
+| source_type   | string  | **Required** | The source type, eg gps or router, of the device. |
```

Comment:
```suggestion
| source_type   | string  | **Required** | The source type, eg GPS or router, of the device. |
```
or
```suggestion
| source_type   | string  | **Required** | The source type, eg `gps` or `router`, of the device. |
```
depending on whether those are the actual set values or just description. Same in the table further below.

Reply:
Let's go with the strings we want in the property.